.gitignore

```diff
@@ -1,5 +1,12 @@
 ### Repo-specific things ###
 
+# Tool Runtime Dir
+/[Tt]ools/
+
+#pkg
+pkg/version.txt
```

Comment:
We should be generating these version files under intermediate now, so somewhere under "bin\obj"